src/qt/trafficgraphwidget.cpp

```diff
@@ -153,12 +148,12 @@ void TrafficGraphWidget::updateRates()
     update();
 }
 
-void TrafficGraphWidget::setGraphRangeMins(int mins)
+void TrafficGraphWidget::setGraphRange(std::chrono::minutes new_range)
```

Comment:
Drop `m_range` and rename this arg to `range`.

Reply:
Unless you early return like `if (m_range == new_range) return`, but I think this won't happen in practice.

Reply:
> Drop `m_range` and rename this arg to `range`.

In a separate (first?) commit, no?

Reply:
Why not the same commit? Just drop `nMin` instead of rename.

Reply:
(a) getting rid of `TrafficGraphWidget::getGraphRangeMins()` and `TrafficGraphWidget::m_range`, and (b) changing type of `TrafficGraphWidget::setGraphRange()` parameter looks really as two focused commits, no?

Reply:
Either way loos fine to me 👍

Reply:
I think this is a valid suggestion. Better to get rid of a redundant member variable.

Addressed in bdf7cb9f0c17bdeeb8389d4b9140aca65e08ac71

@hebasto
> (a) getting rid of TrafficGraphWidget::getGraphRangeMins() and TrafficGraphWidget::nMin, and (b) refactoring of TrafficGraphWidget::setGraphRangeMins() (renaming and parameter type changing) looks really as two focused commits, no?

I agree with your reasoning. However, (a) and (b) are not mutually exclusive, and hence I was having a hard time splitting these two changes while keeping the commits meaningful. Therefore, I reckoned it would be best to make these changes in one commit.